tests/unit/runtime/activation_checkpointing/test_activation_checkpointing.py

```diff
@@ -259,3 +260,52 @@ def test_ckpt_non_tensor_output_ordering(self, non_tensor_output):
         else:
             ordering += [torch.is_tensor(non_tensor_output)]
         _test_activation_checkpoint_ordering(module, ordering, inputs)
+
+
+class TestCheckpointableLayersConfig(DistributedTest):
```

Comment:
Thanks for the PR/test @Quentin-Anthony - looks like the added test is failing, could you take a look?

Reply:
Yep on it

Reply:
Fixed in https://github.com/microsoft/DeepSpeed/pull/6881/commits/b21217739e1d5c21e3572c6ba0ed3b19c53c82d1. Was a silly mistake where I forgot to specify `num_dp=1` for `PipeModelDataParallelTopology`. Test is passing on my end.